easybuild/easyconfigs/l/librsvg/librsvg-2.50.2-foss-2020a.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'librsvg'
+version = '2.50.2'
+
+homepage = 'https://wiki.gnome.org/action/show/Projects/LibRsvg'
+description = """librsvg is a library to render SVG files using cairo."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
```

Comment:
Same here; it can probably go to  GCCcore 10.2.0 (if possible, remember to add the binutils builddep)

Reply:
OK, will do.